<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowTitle">
   <string>Application Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="apiTab">
      <attribute name="title">
       <string>API Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>API Key:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="apiKeyInput">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Endpoint:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="endpointInput"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Timeout (ms):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="timeoutInput">
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>30000</number>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ttsTab">
      <attribute name="title">
       <string>TTS Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_voice">
         <property name="text">
          <string>Default Voice:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="defaultVoiceCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_model">
         <property name="text">
          <string>Default Model:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="modelLayout">
         <item>
          <widget class="QComboBox" name="modelComboBox">
           <property name="toolTip">
            <string>Select quality/latency tradeoff</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshModelsButton">
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/refresh.svg</normaloff>:/icons/refresh.svg</iconset>
           </property>
           <property name="toolTip">
            <string>Refresh available models</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_format">
         <property name="text">
          <string>Output Format:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="formatCombo"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="modelInfoGroup">
         <property name="title">
          <string>Model Information</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_model">
          <item>
           <widget class="QLabel" name="modelDescription">
            <property name="text">
             <string>• tts-1: Fast response, standard quality
• tts-1-hd: Higher quality, moderate latency
• gpt-4o-mini-tts: Advanced features, highest quality</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string>font-size: 11px; color: #666; padding: 5px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>